pkg/services/ngalert/state/compat.go

```diff
@@ -1,4 +1,4 @@
-package schedule
+package state
```

Comment:
`compat` and `compat_test` were moved to the `state` package to get rid of the unnecessary dependency between `api_testing` and `scheduler`.